### PR TITLE
tests(ticdc):  add multi rocks integration test

### LINKED ISSUE #8823: tests(ticdc): add multi rocks integration test
add multi rocks integration test

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: close #8823

#### Tests 

 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: tests/integration_tests/multi_rocks/data/test.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+create database `multi_rocks`;
+use `multi_rocks`;
+
+create table a1 (id int primary key);
+create table a2 (id int primary key);
+create table a3 (id int primary key);
+create table a4 (id int primary key);
+create table a5 (id int primary key);
```
Comment (overvenus) [inline tests/integration_tests/multi_rocks/data/test.sql:8]: Could you use ycsb to load 5 tables and 1000 rows for each?

**Review by overvenus — APPROVED** [review]

**Review by 3AceShowHand — APPROVED** [review]

**Pull request merged**